src/controllers/connectionManager.ts

```diff
@@ -289,6 +289,16 @@ export default class ConnectionManager {
 		return ConnectionCredentials.createConnectionDetails(connectionInfo);
 	}
 
+	/**
+	 * Send a request to the SQL Tools Server client
+	 * @param requestType The of the request to make
```

Comment:
nit: "The type of the request"

Can we make this a bit clearer, by the way?

Reply:
Not sure how much clearer since this is supposed to be generic to be used by many different contract request types